apps/web/src/components/(dashboard)/layout/menu-switcher.tsx

```diff
@@ -102,12 +102,13 @@ export const MenuSwitcher = ({ user, teams: initialTeamsData }: MenuSwitcherProp
             rightSideComponent={
               <ChevronsUpDown className="text-muted-foreground ml-auto h-4 w-4" />
             }
+            showText={false}
           />
         </Button>
       </DropdownMenuTrigger>
 
       <DropdownMenuContent
-        className={cn('z-[60] ml-2 w-full md:ml-0', teams ? 'min-w-[20rem]' : 'min-w-[12rem]')}
+        className={cn('z-[60] ml-4 w-full md:ml-0', teams ? 'min-w-[20rem]' : 'min-w-[12rem]')}
```

Comment:
Just checking, was this part of the design? This was ml-2 to keep the avatars in a straight line

Reply:
> But if it aligns with that, some text is visible which makes it look a bit odd

What do you mean by this? As in the text appears behind it on the left?